packages/libs/react-ui/src/components/Button/Button.tsx

```diff
@@ -7,10 +7,13 @@ import {
 import { SystemIcon } from '@components/Icon';
 import cn from 'classnames';
 import type { ButtonHTMLAttributes, FC } from 'react';
-import React from 'react';
+import React, { type ReactNode } from 'react';
```

Comment:
With the latest change to our eslint rules, which has been merged by now, this will change slightly still so I recommend another format and save on this file after you rebase with main.